### PR TITLE
[feature-1] added new feature

### PR DESCRIPTION
**Pull request by ping2kalesh**
Added new files as part of feature-1 implementation

### CODE REVIEW

File: feature-1/f1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is feature1 code
```
Comment (cmulakala) [inline feature-1/f1.txt]: This looks good, add "." at end.

Comment (cmulakala) [inline feature-1/f1.txt]: Added one more line comment

Comment (sarathgampa) [inline feature-1/f1.txt]: This is looking good.

Comment (ping2kalesh) [reply feature-1/f1.txt]: added . at the end of the line to my code

Comment (ping2kalesh) [reply feature-1/f1.txt]: added . at the end of my line to the code. Could you please review

**Pull request merged**